app/modules/users/models.py

```diff
@@ -154,7 +97,7 @@ def _process_edm_profile_url(self, url):
         log.warning('User._process_edm_profile_url() not implemented yet')
 
 
-class User(db.Model, TimestampViewed, EDMUserMixin):
+class User(db.Model, TimestampViewed, UserEDMMixin):
```

Comment:
I think eventually the user resources will need to be updated to send connected orgs and collaborations as well

Reply:
Yes, the mixins we use can be as complicated as we wish.  I wanted to add the base models so that we can begin to define how they relate to users